### PR TITLE
Fixed a crash when using the high contrast theme

### LINKED ISSUE #645: Folder size is not displayed in the properties dialog
Anotación 2020-04-22 151419

### LINKED ISSUE #1003: App crashes when using High Contrast Theme
App crashes right after the UI is loaded, no error message, and no time frame in which you would be able to click anything.

1. Apply any High Contrast Theme in Windows 10 Settings
2. Start Files UWP from Start List
3. Wait until the UI started loading
4 Watch it close

**Expected behavior**
Expected it to open with High Contrast Theme or at least a normal Theme.

**Desktop (please complete the following information):**
 - Windows 10 1909 18363.778
 - App version: [v0.9.2.0]

**Additional context**
Download from Microsoft Store on 11.June

### PR DESCRIPTION
**Pull request by VoidExploiter**
1. Fixed #645  issue with the help from microsoft support in [this thread](https://docs.microsoft.com/en-us/answers/questions/34670/basicpropertiesretrievepropertiesasyncnew-list-sys.html)

We can also improve the properties dialog by showing file owner, last accessed time like that using 
BasicProperties.RetrievePropertiesAsync()
2. Fixed #1003  It happens there is no known resources for High Contrast Resource Dictionary.

### CODE REVIEW

File: Files/Views/Pages/Properties.xaml.cs [hunk 1]
```diff
@@ -119,7 +129,15 @@ private async void Properties_Loaded(object sender, RoutedEventArgs e)
                 ItemProperties.ItemMD5HashProgressVisibility = Visibility.Collapsed;
             }
         }
-
+        public  string ToBytesCount(long bytes)
```
Comment (tsvietOK) [inline Files/Views/Pages/Properties.xaml.cs:132]: You could use `ByteSizeLib.ByteSize.FromBytes(long).ToString();`

File: Files/Views/Pages/Properties.xaml.cs [hunk 2]
```diff
@@ -54,16 +57,23 @@ private async void Properties_Loaded(object sender, RoutedEventArgs e)
                 if (selectedItem.PrimaryItemAttribute == StorageItemTypes.File)
                 {
                     selectedStorageItem = await StorageFile.GetFileFromPathAsync(selectedItem.ItemPath);
+                    ItemProperties.ItemSize = selectedItem.FileSize;
                 }
                 else if (selectedItem.PrimaryItemAttribute == StorageItemTypes.Folder)
                 {
-                    selectedStorageItem = await StorageFolder.GetFolderFromPathAsync(selectedItem.ItemPath);
+                    var storageFolder = await StorageFolder.GetFolderFromPathAsync(selectedItem.ItemPath);
```
Comment (tsvietOK) [inline Files/Views/Pages/Properties.xaml.cs:64]: Folder size calculation should be async and shouldn't block Properties UI.

File: Files/Views/Pages/Properties.xaml.cs (hunk 1 above)
Comment (VoidExploiter) [reply Files/Views/Pages/Properties.xaml.cs]: Fixed in this [commit](https://github.com/files-community/files-uwp/commit/59fc0f8c336ca5a12a505f43f2071d2e9272990d)

File: Files/Views/Pages/Properties.xaml.cs (hunk 2 above)
Comment (VoidExploiter) [reply Files/Views/Pages/Properties.xaml.cs:64]: Fixed in this [commit](https://github.com/files-community/files-uwp/commit/59fc0f8c336ca5a12a505f43f2071d2e9272990d)

**Pull request merged**